### PR TITLE
lab-site: Initialize lab site

### PR DESCRIPTION
**Pull request by juliaogris**
Initialize lab site and lab.evy.dev subdomain. This is in preparation for a
hands-on lesson with a Melbourne primary school class soon to follow. The lab
subdomain currently hosts a straight up copy of play.evy.dev (symlinks on the
file system). The only difference is the the sample selection intended for
the first, introductory lesson.

### CODE REVIEW

File: frontend/lab/samples/intro/move.evy [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// print 6 circles at x coordinates
+// 0 20 40 60 80 100:
+move 0 50
+circle 10
+
+// 🌶️🌶️
+// Can you change the program to make
+// the circle move from left to right?
+
+// 🌶️🌶🌶
```
Comment (camh-) [inline frontend/lab/samples/intro/move.evy:13]: were the last two meant to be different?

File: build-tools/firebase-deploy [hunk 2]
```diff
@@ -173,7 +173,8 @@ make_pr_urls() {
 	# BASEURL_AI=https://evy-lang-stage--dev-bta34sbm.web.app
 	# BASEURL_DISCORD=https://evy-lang-stage-discord--dev-pap726z0.web.app
 	# BASEURL_DOCS=https://evy-lang-stage-docs--dev-62qb1rk7.web.app
-	# BASEURL_LEARN=https://evy-lang-stage-learn--dev-txkb2kn8.web.app
+	# BASEURL_LAB=https://evy-lang-stage-lab--dev-txkb2kn8.web.app
```
Comment (camh-) [inline build-tools/firebase-deploy:176]: Just double checking that lab was meant to get the suffix that was on learn, and that learn gets a new suffix, not lab.

**Review by camh- — APPROVED** [review]

**Review by camh- — APPROVED** [review]
:broccoli:

File: frontend/lab/samples/intro/move.evy (hunk 1 above)
Comment (juliaogris) [reply frontend/lab/samples/intro/move.evy:13]: nope, fixed, thanks.

File: build-tools/firebase-deploy (hunk 2 above)
Comment (juliaogris) [reply build-tools/firebase-deploy:176]: just demo URLs, but not making sense. Fixed.

**Review by camh- — APPROVED** [review]
:four_leaf_clover:

**Pull request merged**